antora-ui-camel/src/js/04-mobile-navbar.js

```diff
@@ -9,4 +9,21 @@ document.addEventListener('DOMContentLoaded', function () {
       document.documentElement.classList.toggle('is-clipped--navbar')
     })
   })
+
+  var menuDropDowns = Array.prototype.slice.call(document.querySelectorAll('.has-dropdown'), 0)
+  var maxMobileWidth = 1023
+  menuDropDowns.forEach(function (el) {
+    el.addEventListener('click', function (e) {
+      var dropDownMenu = el.querySelectorAll('.navbar-dropdown')[0]
+      if (window.innerWidth <= maxMobileWidth) {
+        if (dropDownMenu.style.display === 'block') {
+          dropDownMenu.style.display = 'none'
+        } else {
+          dropDownMenu.style.display = 'block'
+        }
+      } else {
+        dropDownMenu.style.display = 'none'
+      }
+    })
+  })
```

Comment:
This I'm not to keen on. I'd prefer if we just toggeled the class, the Anotra UI already does. Then we would not have to put the width here but we can have the CSS rules depending on the resolution determine the style. i.e. display can be toggled by CSS rules only.

Reply:
I agree, I hope the changes are fine now. I have toggled the class and added the necessary css to change the display and invert the arrow.